examples/basic_openshift/policy1.yaml

```diff
@@ -0,0 +1,39 @@
+# Filename: policy_for_human_users.yml
```

Comment:
this is inconsistent with the actual policy name, and I like it better than the actual filename (since it's clearer what it contains). can the actual policy name be updated?